Sources/SourceKitLSP/Swift/CodeActions/ConvertIUOToProperOptional.swift

```diff
@@ -0,0 +1,59 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2024 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+import LanguageServerProtocol
+import SwiftRefactor
+import SwiftSyntax
+
+/// Convert implicitly unwrapped optionals (IUOs) to proper optionals
+@_spi(Testing)
+public struct ConvertIUOToProperOptional: SyntaxRefactoringProvider {
```

Comment:
I’d suggest that we’re a little more verbose here and spell out `IUO` as `ImplicitlyUnwrappedOptional`. It’s a little more verbose but makes the code action easier to newcomers to the code base.

Also, let’s just call the `?` syntax *Optional* instead of *Proper Optional*.

```suggestion
public struct ConvertImplicitlyUnwrappedOptionalToOptional: SyntaxRefactoringProvider {
```

Reply:
Agree, I was under a false impression there was a restriction on the maximum length of any file name, as I had never come across a file name longer than this!